core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -245,6 +245,16 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
     private def launchTasks(tasks: Seq[Seq[TaskDescription]]) {
       for (task <- tasks.flatten) {
         val serializedTask = TaskDescription.encode(task)
+        if (serializedTask.limit > TaskSetManager.TASK_SIZE_TO_WARN_KB * 1024) {
```

Comment:
The size of serializedTask exceeds `maxRpcMessageSize` will cause RPC can not send it to executor,  but  the size of serializedTask exceeds `TaskSetManager.TASK_SIZE_TO_WARN_KB` does not cause the problem.
This code is added to keep consistent with the previous code.